projects/gui/src/mainwindow.cpp

```diff
@@ -898,6 +901,16 @@ QString MainWindow::genericTitle(const TabData& gameData) const
 		       .arg(white, black, result.toShortString());
 }
 
+QString MainWindow::nameOnClock(const QString& name, Chess::Side side) const
+{
+	QString text = name;
+	bool displaySide = QSettings().value("ui/display_players_sides_on_clocks", false)
+				      .toBool();
+	if (displaySide)
+		text.append(tr(" (%1)").arg(side.toString()));
```

Comment:
`Side::toString()` already returns a localized string. I think we should just use `QString(" (%1)").arg(side.toString())` here.

Reply:
Thank you. I did not know that and I have not looked it up in the documentation, sorry. I will change this accordingly.